f3/granite.go

```diff
@@ -303,6 +309,12 @@ func (i *instance) isValid(msg *GMessage) bool {
 			return false
 		}
 	}
+
+	sigPayload := SignaturePayload(msg.Instance, msg.Round, msg.Step, msg.Value)
+	if !i.host.Verify(msg.Sender, sigPayload, msg.Signature) {
```

Comment:
This is unrelated, but Verify should probably return an error for debugablity: signature mismatch, sender ID not found, etc.

Reply:
Commented on an issue adding errors to other APIs